configs/nifi.properties

```diff
@@ -147,9 +147,9 @@ nifi.web.jetty.threads=200
 # nifi.web.proxy.context.path=
 
 # security properties #
-nifi.sensitive.props.key=
+nifi.sensitive.props.key={{.Values.properties.sensitiveKey}}
```

Comment:
Not comfortable putting the sensitive key in a nifi-values.yaml file that would be (e.g.) checked into a repository; see my pull request for an alternative approach.